_data/projects/Coverlet.yml

```diff
@@ -0,0 +1,16 @@
+name: Coverlet
+desc: Cross platform code coverage for .NET
+site: https://github.com/tonerdo/coverlet
+tags:
+- cross-coverage
+- dotnet-core
```

Comment:
@MarcoRossignoli Please use Use `.net core` instead of dotnet-core

Reply:
done